accounting/accounting.go

```diff
@@ -125,10 +125,19 @@ func (accounting *State) updateAccountType(addr types.Address, ktype string) {
 func (accounting *State) updateAccountData(addr types.Address, key string, field interface{}) {
 	au, ok := accounting.AccountDataUpdates[addr]
 	if !ok {
-		au = make(map[string]interface{})
+		au = make(map[string]idb.AccountDataUpdate)
 		accounting.AccountDataUpdates[addr] = au
 	}
-	au[key] = field
+	au[key] = idb.AccountDataUpdate{true, field}
+}
+
+func (accounting *State) removeAccountData(addr types.Address, key string) {
+	au, ok := accounting.AccountDataUpdates[addr]
+	if !ok {
+		au = make(map[string]idb.AccountDataUpdate)
+		accounting.AccountDataUpdates[addr] = au
+	}
+	au[key] = idb.AccountDataUpdate{false, struct{}{}}
```

Comment:
Optional - With the struct change, you can also easily merge these two functions to match the struct.